ocfweb/docs/docs/staff/scripts/sorry.md

```diff
@@ -0,0 +1,28 @@
+[[!meta title="sorry: ban/disable an OCF account"]]
+
+## Introduction
+
+`sorry` is the command used to ban or disable OCF accounts. Accounts are banned 
+for various reasons (users/groups being disruptive or otherwise violating 
+OCF/university policies, security issues, lack of contact information, etc.) 
+and alumni sometimes request to have their accounts disabled to stop vhosts, 
+take down information, or for myriad other reasons.
+
+## Usage/Example
+
+    Usage: sorry [user to be sorried] [sorry file]
```

Comment:
Actually I misread what you said, you are correct that the sorry file gets copied there. But that's beside the point; just include the above path somewhere since that's where the default sorry files are.